src/test/java/org/breedinginsight/brapps/importer/GermplasmTemplateMap.java

```diff
@@ -745,7 +746,7 @@ public void checkBasicResponse(JsonObject germplasm, Table fileData, Integer i)
         assertEquals(testUser.getName(), additionalInfo.getAsJsonObject("createdBy").get("userName").getAsString(), "Wrong createdBy userId");
         // Breeding Method
         String breedingMethodName = fileData.getString(i, "Breeding Method");
-        BreedingMethodEntity breedingMethod = breedingMethodDAO.findByNameOrAbbreviation(breedingMethodName).get(0);
+        ProgramBreedingMethodEntity breedingMethod = breedingMethodDAO.findByNameOrAbbreviation(breedingMethodName, validProgram.getId()).get(0);
         assertEquals(breedingMethod.getId().toString(), additionalInfo.get("breedingMethodId").getAsString(), "Wrong Breeding Method ID");
         assertEquals(breedingMethod.getName(), additionalInfo.get("breedingMethod").getAsString(), "Wrong Breeding Method name");
```

Comment:
This is preexisting code but, it would better if it used the public static finals BrAPIAdditionalInfoFields.GERMPLASM_BREEDING_METHOD_ID and 
BrAPIAdditionalInfoFields.GERMPLASM_BREEDING_METHOD instead of the hard coded "breedingMethodId" and "breedingMethodId"

Reply:
Done